src/chrome/content/rules/www.2chan.net.xml

```diff
@@ -0,0 +1,14 @@
+<ruleset name="www.2chan.net">
```

Comment:
Don't name the ruleset "www.2chan.net.xml" rather "2chan.net.xml"

Reply:
Thanks for your input!

I am aware that the **www** prefix in the rules directory is not the way to go when it comes to creating new rulesets, so the only reason for that ruleset name was because 2chan.net is unreachable which can be confirmed by executing `ping 2chan.net` which results in:

> ping: 2chan.net: No address associated with hostname

However, it's fixed now.